dotnet/src/Skills/Skills.OpenAPI/Extensions/KernelOpenApiExtensions.cs

```diff
@@ -262,7 +262,7 @@ public static class KernelOpenApiExtensions
     private static ISKFunction RegisterRestApiFunction(
         this IKernel kernel,
         string skillName,
-        IRestApiOperationRunner runner,
+        RestApiOperationRunner runner,
```

Comment:
Are you thinking we just remove the IRestApiOperationRunner interface altogether?

Reply:
I wasn't, though I don't have an objection to doing so. I changed this as it's private and an analyzer was warning that all the callers were the concrete type so there was no benefit and only cost to this taking the interface.